client/app/components/Input/Input.scss

```diff
@@ -223,3 +223,33 @@
 .hidden {
   display: none;
 }
+
+.show {
+  margin-top: $size-60;
+  @media screen and (max-width: 1035px) and (min-width: 449px)
```

Comment:
We should be using the variables we have for our breakpoints. Please see: `app/assets/stylesheets/base/_breakpoints.scss`.

Reply:
I also recommend moving these changes to `InputPassword.scss`. We'll also want to offset this show button in a way that makes it consistent with our existing padding.